items.json

```diff
@@ -15252,9 +15252,9 @@
     "name": "Heavy Flanged Mace",
     "culture": "Khuzait",
     "type": "OneHandedWeapon",
-    "price": 198,
-    "weight": 2.29,
-    "tier": 0.7856775,
+    "price": 2020,
+    "weight": 1.97,
+    "tier": 4.66271448,

```

Comment:
is this supposed to be a mace for beginners? If the answer is yes i'm fine with it, just want to double check